astroquery/oac/core.py

```diff
@@ -479,22 +479,20 @@ def _parse_result(self, response, verbose=False):
 
         try:
```

Comment:
there is no need for this try except any more, please remove.

Reply:
Done